addons/mail/static/tests/chatter_tests.js

```diff
@@ -8,6 +8,7 @@ var concurrency = require('web.concurrency');
 var FormView = require('web.FormView');
 var KanbanView = require('web.KanbanView');
 var testUtils = require('web.test_utils');
+var systray = require('mail.systray');
```

Comment:
mail.systray is before web.FormView in alphabetic order, move this before the FormView import